### DISCUSSION TITLE
PylonCapture not working

### DISCUSSION THREAD

**Discussion by aniachr3** [bonsai-rx/bonsai#1553] [Q&A]
Hi there!

I have an issue with PylonCapture. After updating Bonsai to the newest version, I cannot run the node at all (not even in a new workflow), I'm getting the following error message: "External component has thrown an exception". Pylon viewer displays the camera image correctly. Any idea what could be the source of the problem?

Thanks in advance!
Ania

**Comment by bruno-f-cruz** [comment]
It seems Bonsai is not able to find the dependencies to run the package. Prior to the update, did you manage to acquire data using the same exact configuration (camera, computer, pylon version, etc...)?

What is the version of the package and of the SDK you are using?

**Reply by aniachr3** [reply]
Today before updating the acquisition was ok. I had pylon Viewer version: 5.2.0. After your comment I saw that PylonCapture requires pylon 7.1.0, so I installed it, and now I'm getting a different error: "The type initializer for 'OpenCV.Net.Native.NativeMethods' threw an exception. file 'streamgrabberimpl.cpp', line 469"

**Reply by bruno-f-cruz** [reply]
How did you install bonsai? From a portable version or system-wide?

**Reply by aniachr3** [reply]
system-wide

**Reply by bruno-f-cruz** [reply]
Could you run a clean install with the latest version available on the website? I suspect you don't have a opencv dependency. If that doesn't work can you try to install this: https://www.microsoft.com/en-us/download/details.aspx?id=30679

**Reply by aniachr3** [reply]
I will do that, thanks! I quickly checked running PylonCapture on a portable Bonsai version and it works well.

update: clean install solved the problem
